pexpect/popen_spawn.py

```diff
@@ -153,6 +152,10 @@ def kill(self, sig):
 
         os.kill(self.proc.pid, sig)
 
+    def sendeof(self):
+        if sys.platform == 'win32':
+            self.proc.stdin.close()
```

Comment:
It also doesn't seem right for this to be a no-op on Unix. What's wrong with closing stdin() regardless of platform?